rift/node.py

```diff
@@ -2378,51 +2400,65 @@ def defer_spf_timer_expired(self):
             self.spf_run()
 
     def ties_of_type(self, direction, system_id, prefix_type):
-        # Return an ordered list of TIEs from the given node and in the given direction and of the
-        # given type
+        # Return an ordered list of TIEs from the given node and in the given direction and of the given type
         node_ties = []
+
         start_tie_id = packet_common.make_tie_id(direction, system_id, prefix_type, 0)
-        end_tie_id = packet_common.make_tie_id(direction, system_id, prefix_type,
-                                               packet_common.MAX_U32)
+        end_tie_id = packet_common.make_tie_id(direction, system_id, prefix_type, packet_common.MAX_U32)
+
         node_tie_ids = self.tie_packet_infos.irange(start_tie_id, end_tie_id, (True, True))
+
         for node_tie_id in node_tie_ids:
             node_tie_packet_info = self.tie_packet_infos[node_tie_id]
             node_tie_packet = node_tie_packet_info.protocol_packet.content.tie
             node_ties.append(node_tie_packet)
+
         return node_ties
 
     def node_ties(self, direction, system_id):
         # Return an ordered list of all node TIEs from the given node and in the given direction
         return self.ties_of_type(direction, system_id, common.ttypes.TIETypeType.NodeTIEType)
 
-    def node_neighbors(self, node_ties, neighbor_direction):
+    def node_neighbors(self, node_ties, neighbor_directions):
         # A generator that yields (nbr_system_id, nbr_tie_element) tuples for all neighbors in the
-        # specified direction of the nodes in the node_ties list.
+        # specified directions of the nodes in the node_ties list.
         for node_tie in node_ties:
             node_level = node_tie.element.node.level
+
```

Comment:
Avoid blank lines in short functions

PS: I am going to stop putting this comment on every new blank line, please apply the same comment to all new added blank lines from here on.  Sorry if I am being pedantic, but consistent code style is important to me.

The general rule is that if the function is short without any comments inside the function body, don't insert blank lines.

If the function is long, consisting of several functional blocks, then each block should have a comment explaining what the block does, and there should be one blank line between each block.